llama-rs/src/lib.rs

```diff
@@ -823,18 +875,24 @@ impl Model {
         {
             let scale = 1.0 / params.temp;
             for (i, &logit) in logits.iter().enumerate() {
+                let tid = i as TokenId;
+                let bias = params.bias_tokens.get(tid);
+
                 // repetition penalty from CTRL paper (https://arxiv.org/abs/1909.05858)
                 // credit https://github.com/facebookresearch/llama/compare/main...shawwn:llama:main
-                if session.last_n_tokens.contains(&(i as TokenId)) {
+                let val = if bias != 0.0 {
+                    bias
+                } else if session.last_n_tokens.contains(&tid) {
```

Comment:
Calling it `bias` sounds a bit weird. Generally a bias is something that is added to the original value. But here, the bias entirely replaces the value?

Reply:
I called it that based on the `logit_bias` field in the TextSynth API: https://textsynth.com/documentation.html#completions

You can call it whatever you want (except late for dinner!) What would you prefer the name to be?